pam_tcb/support.h

```diff
@@ -147,10 +147,19 @@ struct unix_verify_password_param {
 	const char *pass;
 };
 
+#define pam_tcb_overwrite_string(xx)					\
+do {									\
+	void *xx__ = xx;						\
+	if (xx__) {							\
+		xx__ = memset(xx__, '\0', strlen(xx__));		\
+		__asm__ __volatile__ ("" : : "r"(xx__) : "memory");	\
+	}								\
+} while (0)
```

Comment:
I don't mind the `do` / `while (0)` trick, but note that `_pam_delete` below does not use it - so maybe also don't use it here for consistency?

Reply:
OK, let's keep it simple for consistency.